### PR TITLE
Add sqrt

### PR DESCRIPTION
**Pull request by mschauer**
Solves #18

### CODE REVIEW

File: src/LogarithmicNumbers.jl [hunk 1]
```diff
@@ -605,6 +605,10 @@ function Base.exp(x::Logarithmic)
     x.signbit ? inv(exp(x.abs)) : exp(x.abs)
 end
 
+function Base.sqrt(x::AnyLogarithmic)
+    uexp(x.log/2)
```
Comment (cjdoris) [inline src/LogarithmicNumbers.jl:609]: please change to `uexp(x.log / oftype(x.log, 2))` in order to preserve the type of `x.log`

**Review by cjdoris — CHANGES_REQUESTED** [review]
Thanks for this! Could you add a test case too please - you could adapt `@testcase "pow"`.

File: src/LogarithmicNumbers.jl (hunk 1 above)
Comment (mschauer) [reply src/LogarithmicNumbers.jl:609]: I can change it, but division by 2 is generally typesafe

**Comment by cjdoris** [comment]
Thanks very much!

**Pull request merged**